packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -160,11 +158,11 @@ class FuchsiaDevices extends PollingDeviceDiscovery {
     required FuchsiaWorkflow fuchsiaWorkflow,
     required FuchsiaSdk fuchsiaSdk,
     required Logger logger,
-  }) : _platform = platform,
-       _fuchsiaWorkflow = fuchsiaWorkflow,
-       _fuchsiaSdk = fuchsiaSdk,
-       _logger = logger,
-       super('Fuchsia devices');
+  })  : _platform = platform,
```

Comment:
Please revert this style only nit per: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

Reply:
done